dashboard/alerts.html

```diff
@@ -6,14 +6,18 @@
 </head>
 <body>
   <div ng-app="alertsApp" ng-controller="alertsCtrl">
-    <button ng-click="alert()">Good Play!</button>
+    <button ng-click="alertGoodPlay()">Good Play!</button>
+    <button ng-click="alertBadLie()">Bad Lie!</button>
```

Comment:
The anti-matter to repetition is abstraction.  One very simple layers of abstraction that we could put in here would be to abstract out the "message" that gets sent.  The message could be a parameter to the "alert" function, such that rather than calling "alertGoodPlay" and "alertBadLie", we could just have the ng-click values be "alert('goodPlay')" and "alert('badLie')".  That way, we would only need one function.  Something like:

```
$scope.alert = function(message) {
  nodecg.sendMessage(message);
}
```

We could further abstract if we used ng-repeat on an object that looked like:
```
{
  'goodPlay': {
    message: 'goodPlay',
    button: 'Good Play!'
  }
  'badLie': {
    message: badLie,
    button: 'Bad Lie!'
  }
}
```

Then, any time we want to add another button/alert, we just need to add an entry to that object.  Ideally, adding an entry to that object in the constants.js file would be enough to add a new button in the dashboard and alert in the graphics.

Reply:
This makes sense and is SUPER helpful <3